website/content/docs/configuration/storage/raft.mdx

```diff
@@ -51,20 +51,20 @@ set [`disable_mlock`](/vault/docs/configuration#disable_mlock) to `true`, and to
 - `node_id` `(string: "")` - The identifier for the node in the Raft cluster.
   This value can be overridden by setting the `VAULT_RAFT_NODE_ID` environment variable.
 
-- `performance_multiplier` `(integer: 0)` - An integer multiplier used by
+- `performance_multiplier` `(integer: 0)` - An integer multiplier used by 
   servers to scale key Raft timing parameters. Tuning this affects the time it
   takes Vault to detect leader failures and to perform leader elections, at the
   expense of requiring more network and CPU resources for better performance.
-  Omitting this value or setting it to 0 uses default timing described below.
-  Lower values are used to tighten timing and increase sensitivity while higher
-  values relax timings and reduce sensitivity.
-
-  By default, Vault will use a lower-performance timing that's suitable for
-  minimal Vault servers, currently equivalent to setting this to a value of 5
-  (this default may be changed in future versions of Vault, depending if the
-  target minimum server profile changes). Setting this to a value of 1 will
-  configure Raft to its highest-performance mode and is recommended for
-  production Vault servers. The maximum allowed value is 10.
+  Omitting this value or setting it to 0 uses default timing described below. 
+  Lower values represent higher frequencies with lower intervals while lower 
```

Comment:
"Lower values ... while lower values ... " seems wrong, was that change intentional?  Also, you're speaking about "frequencies" and "intervals" but without having introduced those terms; what frequency, what interval are we speaking about?  I feel the original phrasing in this paragraph was clearer than the replacement you're suggesting.

Reply:
I have updated the first paragraph with original text. Thank you !